.devcontainer/python-3.10/docker-compose.yml

```diff
@@ -22,6 +22,8 @@ services:
     # ports:
       # - "${PORT_PREFIX}4222:4222"
       # - "${PORT_PREFIX}9222:9222"
+    env_file:
```

Comment:
why is this needed now?

Reply:
In local also I used to run NATS without auth. 
Now I am setting up auth in local NATS using config file - https://github.com/airtai/fastagency/pull/278/files#diff-c13c0a1a0400d35b0786cde52030de4cdbaa32fb6df1e5a0584546b0ccf524af. 
In the config file I am using env variables to setup password for NATS users so I need to pass env file to NATS in order to set those env vars - https://github.com/airtai/fastagency/pull/278/files#diff-ddd7709d62c48ce521c3efe76e09432f4ce1aceefebb93b5353973c9dddbdb06

In staging and prod, I am not going to hardcode password instead going to use env variable. I have replicated the same approach in local devcontainer.